setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = aiohttp-cors
+version = 0.7.0
+author = Vladimir Rutsky and aio-libs team
+author_email = vladimir@rutsky.org
+license = Apache-2.0
+description = CORS support for aiohttp
+long_description = file: README.rst
+url = https://github.com/aio-libs/aiohttp-cors
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Topic :: Software Development :: Libraries
+    Topic :: Internet :: WWW/HTTP
+    Framework :: AsyncIO
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    License :: OSI Approved :: Apache Software License  
+    Development Status :: 3 - Alpha
+
+[options]
+packages = aiohttp_cors
+python_requires = >=3.4.1
+install_requires =
+    aiohttp>=1.1
+    typing;python_version<'3.5'
+setup_requires =
+    # Environment markers were implemented and stabilized in setuptools
+    # v20.8.1 (see <http://stackoverflow.com/a/32643122/391865>).
+    setuptools>=20.8.1
+    # If line above doesn't work, check that you have at least
+    # setuptools v19.4 (released 2016-01-16):
+    # <https://github.com/pypa/setuptools/issues/141>
+
+tests_require =
```

Comment:
What is the reason to keep unpinned test dependencies here?

Reply:
Just moved from https://github.com/aio-libs/aiohttp-cors/pull/212/files/6cd0705e2b3924b263a0305f7562f27d2f355f5b#diff-2eeaed663bd0d25b7e608891384b7298L60